### ISSUE TITLE
Automated image converter works not well

### ISSUE THREAD

**Issue by Techming** [jmoenig/Snap--Build-Your-Own-Blocks#2150]
Each time I import a custom image( whether for stage or costume), and I save the project, all costumes and stage images are pixelated. This situation will not occur until I save my project so I think maybe Snap! convert imported image into a smaller format. The low quality image really makes the whole stage looks bad. Can anyone try to fix this problem? THX

**Comment by jguille2** [comment]
Hi @Techming ,

I can not reproduce your issue. Could you share your _custom image_ to test it?

**Comment by Techming** [comment]
Hi @jguille2 ，
Here is my custom image
![hw3_testonly](https://user-images.githubusercontent.com/41393704/43017486-27fb26ee-8c0b-11e8-9540-d607cb4b4db6.png)
And there are some screen shot for showing the problem (in order from left to right)
<img width="1435" alt="2018-07-20 10 46 43" src="https://user-images.githubusercontent.com/41393704/43017338-93748060-8c0a-11e8-9eb8-a5dc05e0850e.png">
 1.Original image (Exported with resolution 300, same size with Snap! stage. 
 2.Screen shot from Snap! when the custom image imported but the project file hasn't been saved.
 3.Screen shot from Snap! when the custom image imported and the project file is saved.
Also screen shot from Snap! in Full screen mode
<img width="1440" alt="2018-07-20 10 46 53" src="https://user-images.githubusercontent.com/41393704/43017433-fb61da88-8c0a-11e8-8a7e-55466977177e.png">

**Comment by cycomachead** [comment]
Does your computer have a high resolution / retina display? (Not sure yet if that's related, but it's possible.)

**Comment by Techming** [comment]
@cycomachead 
Yes. I'm using MBP to do my project. Maybe I should use Win to test if the same problem exists. 
Thank you.

**Comment by Techming** [comment]
@cycomachead I tried to import the same image into Snap! running on a windows OS, but the same issue happened again. The image was pixelated.

**Comment by jmoenig** [comment]
images become "normalized" when turned into costumes. Hi-res stuff from retina displays puts a big strain on performance. Problem is, that Mac retina displays totally suck at displaying normal images (I can tell, because I'm using a MBP myself). The same image on a non-retina display looks quite well. Bottom line: Yes, I'm downsizing images and normalizing them for costumes.
